### ISSUE TITLE
No mention of valid range of SSID values

### ISSUE THREAD

**Issue by mr-intj** [MicrosoftDocs/winrt-api#2136]
There's not much information here (except for the mention that it should be ASCII). I was specifically interested in what it might mean to set the SSID to an empty string. Would that be an error? Would it effectively disable SSID broadcast?

It's possible to Google for the max SSID length, since there's a standard. The answer seems to be 32, but it would be more time-efficient if one docs writer had added that info rather than every developer perpetually having to independently look it up. 

Microsoft has made many efforts through the decades to improve the stability of Windows apps (and reduce security vulnerabilities). Like a lot of my peers, I've embraced those efforts. It's not fair that Microsoft gets the blame when poorly-written third-party software crashes on their OS. Over the decades, I've tried to develop good habits for writing robust code, checking preconditions and return values, etc.

But it's unnecessarily difficult to write solid code when so many of Microsoft's APIs have such minimal documentation (e.g., ranges of function arguments not being mentioned at all). I realize that it's a big job to catch up on all of this, but please realize how massive the overhead is when conscientious Windows developers all around the world keep having to switch to empirically testing some under-documented API to try to fill in the gaps in the docs.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c7c49f3f-cae7-d748-f83e-d55172cac076
* Version Independent ID: b1559ab1-29fb-1068-4962-c22b53e88af7
* Content: [NetworkOperatorTetheringAccessPointConfiguration.Ssid Property (Windows.Networking.NetworkOperators) - Windows UWP applications](https://docs.microsoft.com/en-us/uwp/api/windows.networking.networkoperators.networkoperatortetheringaccesspointconfiguration.ssid?view=winrt-22000)
* Content Source: [winrt-api-build/xml/Windows.Networking.NetworkOperators/NetworkOperatorTetheringAccessPointConfiguration.xml](https://github.com/MicrosoftDocs/winrt-api/blob/docs/winrt-api-build/xml/Windows.Networking.NetworkOperators/NetworkOperatorTetheringAccessPointConfiguration.xml)
* Product: **uwp**
* Technology: **winrt-reference**
* GitHub Login: @stevewhims
* Microsoft Alias: **stwhi**

**Comment by stevewhims** [comment]
Hi, thanks for filing this issue. I've reached out to the feature team for advice.

-Steve

**Comment by stevewhims** [comment]
Hi, @mr-intj.

I made a correction to this topic. Previously the description said that the SSID property returns the network account ID. In fact, it returns the SSID to be used for the tethering network. I'm still trying to get info about your specific questions.

Thanks!
Steve

**Comment by mr-intj** [comment]
Thanks Steve.

Apologies for my snippy tone. I run into this pretty often, so by the time I'm motivated enough to post feedback...

**Comment by stevewhims** [comment]
UPDATE: I just sent another reminder email internally, since I'd like to get answers to your question about setting the SSID to an empty string, and confirm that its length really is 32 ASCII chars.

**Comment by stevewhims** [comment]
I've published an update to the topic. Here's the paragraph I added to describe the property value:

A string containing the SSID to be used for the tethering network. An SSID is the name of the Wi-Fi network (as would be displayed in the list of nearby networks, for example). An SSID must be between 1 and 32 characters in length; and the first and last character may not be a space.

I'll go ahead and close the issue.

Thanks!
Steve
